app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+class CpError(Exception):
+    pass
+
+
+def copy_file(command: str) -> None:
+    files = command.split()
```

Comment:
The variable name `files` is not descriptive. Consider renaming it to something more meaningful, like `command_parts`, to improve code readability.